src/panels/lovelace/elements/hui-state-label-element.ts

```diff
@@ -70,13 +90,9 @@ class HuiStateLabelElement extends LitElement implements LovelaceElement {
           hasAction(this._config.tap_action) ? "0" : undefined
         )}
       >
-        ${this._config.prefix}${stateObj
-          ? computeStateDisplay(
-              this.hass.localize,
-              stateObj,
-              this.hass.language
-            )
-          : "-"}${this._config.suffix}
+        ${this._config.prefix}${attribute
+          ? attributeDisplay
```

Comment:
Best practice for ifs/ternary is for the condition that is expected to be true most of the time should be in the if section while the rare condition in the else section. So for this I would do ! attribute then state else attribute